main.py

```
@@ -1,23 +1,9 @@
-n = 5
-result = nth_fibonacci(n)
-print(result)
-
-
 # Function to calculate the nth Fibonacci number using memoization
 def nth_fibonacci_util(n, memo):
 
     # Base case: if n is 0 or 1, return n
     if n <= 1:
-        return n
+        return m
```

Ruff (0.12.2)

Undefined name `m`

(F821)